package.json

```diff
@@ -293,6 +293,22 @@
                             "buildLog": {
                                 "type": "string",
                                 "description": "%makefile-tools.configuration.makefile.configurations.buildLog.description%"
+                            },
+                            "preConfigureArgs": {
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                },
+                                "default": [],
+                                "description": "%makefile-tools.configuration.makefile.preConfigureArgs.description%"
+                            },
+                            "postConfigureArgs": {
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                },
+                                "default": [],
+                                "description": "%makefile-tools.configuration.makefile.postConfigureArgs.description%"
                             }
                         }
```

Comment:
Missing pre/post configure script at configuration level. We have args at configuration level and script+args at global level but I don't see script at config level.

Reply:
This was a design decision I made, but I'm happy to discuss whether it was the right one. 

My thoughts were that, this would allow users to have pre/post configure scripts at the global level, and they can use arguments to allow it to be per configuration. My estimation was that people would rather do that then to have an actual per configuration script.

Reply:
Oh, that is an alternative indeed. I just started from the idea of having per config scripts as well but I can see how having one script and per config args can work the same. If we find in future that users actually have different content of commands and script paragraphs, we can still add it later without breaking this design and the users that will start using it this way. I'm good as it is.